<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>361</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>461</width>
    <height>361</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>461</width>
    <height>361</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Encounter</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="lblDimerBasis">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Dimer Basis</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMonomerBasis">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Monomer Basis</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblDimerDimer">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>74</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Dimer</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMoleculeEnergy">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Molecule Energy /au</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtDimerDimer">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>70</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMonADimer">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>100</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMonBDimer">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>130</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblMonADimer">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>104</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Monomer A</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMonBDimer">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>134</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Monomer B</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMonAMon">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>200</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtMonBMon">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>230</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblMonAMon">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>205</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Monomer A</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblMonBMon">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>235</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Monomer B</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblInteractionEnergy">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Interaction Energy</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtInteractionHartree">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>70</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtInteractionKjmol">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>100</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lblInteractionHartree">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>75</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>/Ha</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblInteractionKJmol">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>105</y>
      <width>41</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>/kJ/mol</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblBindingConstant">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>170</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Binding Constant</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblNoUnits">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>205</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>/1</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtBindingConstant">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>200</y>
      <width>113</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>461</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen"/>
   <addaction name="actionExport"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/open.png</normaloff>:/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="icon">
    <iconset resource="icons.qrc">
     <normaloff>:/icons/save.png</normaloff>:/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Export...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
